idx/memory/tag_query_test.go

```diff
@@ -483,55 +487,6 @@ func TestGetByTag(t *testing.T) {
 	}
 }
 
-func TestDeleteTaggedSeries(t *testing.T) {
```

Comment:
why is this removed? is it replaced by something else? if so, what?

Reply:
The interface to delete tagged series has changed. Previously `Delete()` could be used to delete tagged series, while now those methods have been separated into `Delete()` and `DeleteTagged()`. There are other tests for `DeleteTagged()`, so this one is not necessary anymore.